hal/armv7m/stm32/l4/timer.c

```diff
@@ -161,10 +161,18 @@ time_t hal_timerGetCyc(void)
 
 int hal_timerRegister(int (*f)(unsigned int, cpu_context_t *, void *), void *data, intr_handler_t *h)
 {
+	int err;
+
 	h->f = f;
 	h->n = lptim1_irq;
```

Comment:
`lptim1_irq` is registered in `hal_timerInit`, so I think that mistake is related only to wrong irq number.
What do you think, about changing only irq number to `SYSTICK_IRQ` instead of registering two handlers?

Reply:
You're right, changed, thx